### PR TITLE
Assignment7 MattJett-EWU and twofingerrightclick

### PR DESCRIPTION
**Pull request by twofingerrightclick**
working with @MattJett-EWU

### CODE REVIEW

File: SecretSanta/src/SecretSanta.Web/scripts/list-Gifts.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// Unit Tests
+describe('getAllGifts', () => {
+    it('should return one gift, being id of 42, and userId of 3', async () => {
+        const gifts = await new GiftList(new MockGiftClient()).getAllGifts();
+        expect(gifts.length).to.equal(1);
+        expect(gifts[0].id).to.equal(999);
+        expect(gifts[0].userId).to.equal(3);
+    });
+});
+
```
Comment (MattJett-EWU) [inline SecretSanta/src/SecretSanta.Web/scripts/list-Gifts.spec.ts]: I think we need to add more unit tests, we'll have to fix that later

File: SecretSanta/src/SecretSanta.Web/Controllers/HomeController.cs [hunk 2]
```diff
@@ -12,6 +12,11 @@ public class HomeController : Controller
     {
         // GET: /<controller>/
         public IActionResult Index()
+        {
+            return View("~/Views/Gifts/ListGifts.cshtml");
```
Comment (twofingerrightclick) [inline SecretSanta/src/SecretSanta.Web/Controllers/HomeController.cs]: @MattJett-EWU this is where the homepage is set

File: SecretSanta/src/SecretSanta.Web/Startup.cs [hunk 3]
```diff
@@ -24,7 +24,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddHttpClient("SecretSantaApi", options =>
             {
                 options.BaseAddress = new Uri(Configuration["ApiUrl"]);
-            });
+            });
+
+
+           /* services.AddCors(options =>
```
Comment (twofingerrightclick) [inline SecretSanta/src/SecretSanta.Web/Startup.cs]: cors goes in api startup... comment can be removed

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width" />
     <title>@ViewBag.Title</title>
     <!-- Stylesheets -->
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
```
Comment (twofingerrightclick) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml]: webpack defaults to favicon

Comment (twofingerrightclick) [reply SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml]: for the name. cannot change the name of the favicon file

File: SecretSanta/src/SecretSanta.Web/scripts/list-Gifts.ts [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+       
+
+        var gifts = await this.getAllGifts();
+        console.log(`number of gifts: ${gifts.length}`);
+
+        var tableBody = document.getElementById("tableBody");
+       
+
+
```
Comment (twofingerrightclick) [inline SecretSanta/src/SecretSanta.Web/scripts/list-Gifts.ts]: this script sometimes often has an array of gifts fromt he client.getll call tht is larger than the number of gifts actually in the database... confused about this

**Pull request closed without merging**